src/app.module.ts

```diff
@@ -21,9 +21,20 @@ import { FirebaseAdminCoreModule } from './firebase/firebase-admin.module'
 import { ResultsModule } from './results/results.module'
 import { TranslateStatusInterceptor } from './i18n/translate-status.interceptor'
 import { CommandModule } from 'nestjs-command'
+import { GoogleRecaptchaModule } from '@nestlab/google-recaptcha'
 
 @Module({
   imports: [
+    GoogleRecaptchaModule.forRootAsync({
+      imports: [ConfigModule],
+      inject: [ConfigService],
+      useFactory: (configService: ConfigService) => ({
+        score: 0.8,
+        secretKey: configService.get('GOOGLE_RECAPTCHA_SECRET_KEY'),
+        response: (req) => req.headers['x-recaptcha-token'] ?? '',
+        skipIf: process.env.NODE_ENV !== 'production',
```

Comment:
Let's move this to an explicit boolean env var `GOOGLE_RECAPTCHA_ENABLED` with a default of `false`.